browser_use/agent/prompts.py

```diff
@@ -288,6 +292,7 @@ def get_user_message(self, use_vision: bool = True) -> UserMessage:
 						image_url=ImageURL(
 							url=f'data:image/png;base64,{screenshot}',
 							media_type='image/png',
+							detail=self.vision_detail_level,
```

Comment:
I don't think this arg exists? https://ai.pydantic.dev/api/messages/#pydantic_ai.messages.ImageUrl